src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -264,7 +266,23 @@ class BinaryCompactObject : public DomainCreator<3> {
         "The time dependence of the moving mesh domain."};
   };
 
-  using options = tmpl::list<
+  template <typename BoundaryConditionsBase>
+  struct InnerBoundaryCondition {
+    static std::string name() noexcept { return "InnerBoundaryCondition"; }
```

Comment:
For consistency with Interval et al, return "InnerBoundary" here ... or actually maybe "ExcisionBoundaries" is even better, to make clear it'll apply to any/all excision regions if present?